requirements-test.txt

```diff
@@ -141,10 +141,6 @@ idna==2.8 \
     --hash=sha256:c357b3f628cf53ae2c4c05627ecc484553142ca23264e593d327bcde5e9c3407 \
     --hash=sha256:ea8b7f6188e6fa117537c3df7da9fc686d485087abf6ac197f9c46432f7e4a3c \
     # via requests
-importlib-metadata==1.5.0 \
```

Comment:
It looks like this should've been left behind, and not removed:
```
In --require-hashes mode, all requirements must have their versions pinned with ==. These do not:
    importlib-metadata>=0.18; python_version < "3.8" from https://files.pythonhosted.org/packages/ad/e4/891bfcaf868ccabc619942f27940c77a8a4b45fd8367098955bb7e152fb1/importlib_metadata-1.6.0-py2.py3-none-any.whl#sha256=2a688cbaa90e0cc587f1df48bdc97a6eadccdcd9c35fb3f976a09e3b5016d90f (from kombu==4.6.7->-r requirements-test.txt (line 152))
```

Reply:
Created https://github.com/release-engineering/iib/pull/83

Reply:
Thanks Luiz! I'm going to go ahead and close this PR